packages/web-component/src/components.d.ts

```diff
@@ -6,7 +6,6 @@
  */
 import { HTMLStencilElement, JSXBase } from "@stencil/core/internal";
 import { InterfaceLanguage, ReadAlongMode } from "./index.d";
-export { InterfaceLanguage, ReadAlongMode } from "./index.d";
```

Comment:
AP's recent PR had this line automatically inserted, now this PR automatically removes it. I wonder if we might be working with different versions of node or something. Did you recently redo `npm install` in the project?